src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write code here
+
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const horizontalPosition = (wall.clientWidth - spider.clientWidth) / 2;
+
+spider.style.left = horizontalPosition + 'px';
+
+const verticallyPosition = (wall.clientHeight - spider.clientHeight) / 2;
+
+spider.style.top = verticallyPosition + 'px';
```

Comment:
Your wall is square so height and width are the same. so you can simplify this code and get rid of duplicates